src/taco-cli/cli/utils/templateManager.ts

```diff
@@ -236,40 +186,32 @@ class TemplateManager {
             });
     }
 
-    private acquireTemplate(templateId: string, kitId: string): Q.Promise<string> {
+    private acquireTemplate(templateId: string, kitId: string, tempTemplateDir: string = os.tmpdir()): Q.Promise<string> {
```

Comment:
Unit tests need to customize where the temporary templates are extracted. I figured we didn't want to pollute the machines with testing artifacts. TemplateManager.ts tests use the test run folders as the parent dir for temporary templates.

What alternate design would you propose, with that in mind?